src/components/swapv2/AdvancedSwapDetails.tsx

```diff
@@ -306,15 +306,15 @@ export function TradeSummaryCrossChain({
 }) {
   const theme = useTheme()
   const navigate = useNavigate()
-  const { chainId } = useActiveWeb3React()
+  const { chainId, account } = useActiveWeb3React()
 
   const [show, setShow] = useState(true)
   const { duration, minReceive, priceImpact, totalFeeUsd, gasFeeUsd, crossChainFeeUsd } = getRouInfo(route)
 
   const nativeToken = NativeCurrencies[chainId]
   const { isEnoughEth, gasFee, crossChainFee } = useIsEnoughGas(route)
 
-  const enoughGasFee = !route ? true : isEnoughEth
+  const enoughGasFee = !route || !account ? true : isEnoughEth
```

Comment:
this should be a part of the `useIsEnoughGas` hook. 
isEnoughEth should include the route and account check

Reply:
make sense. updated